packages/kbn-optimizer/limits.yml

```diff
@@ -117,7 +117,7 @@ pageLoadAssetSize:
   securitySolution: 66738
   serverless: 16573
   serverlessObservability: 16582
-  serverlessSearch: 22555
+  serverlessSearch: 232448
```

Comment:
Looks like we upped this 10x, typo somewhere?

Reply:
Yeah, went wrong in fixing a merge conflict. Updated in latest :)